docs/notebooks/Ranking_by_Reordering.ipynb

```diff
@@ -90,9 +81,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The goal is then to find the optimal permutation, $\\mathbf{Q}$, that maximizes the similarity between the rank-differential matrix, $\\mathbf{R}$, and the score-differential matrix, $\\mathbf{D}$. This is a two-sided permutation Procrustes problem with one transformation,\n",
+    "The rank-differential matrix $\\mathbf{R} \\in \\mathbb{R}^{n \\times n}$ is an upper-triangular matrix and its $ij$-th element specifies the difference in ranking between team $i$ and team $j$. This a sensible target for the score-differential matrix. Now, the two-sided permutation Procrustes method can be used to find the permutation matrix that maximizes the similarity between the score-differential matrix, $\\mathbf{A}$, and the rank-differential matrix based on the definition of rank-differential matrix, $\\mathbf{B}$ (**Fig. (ii)**)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
     "\\begin{equation}\n",
-    "    \\min_{\\mathbf{Q}} {\\left\\lVert \\mathbf{Q}^{\\top} \\mathbf{D} \\mathbf{Q} - \\hat{\\mathbf{R}}\n",
+    "    \\min_{\\mathbf{P}} {\\left\\lVert \\mathbf{P}^{\\top} \\mathbf{A} \\mathbf{P} - \\mathbf{B}\n",
```

Comment:
We are not including the formula for any of the examples, so we should either add formulas for all or remove this. Alternatively, we can add a link to the documentation. What do you think @PaulWAyers?

Reply:
Good point @PaulWAyers. Let's add equations to all notebooks then.